### ISSUE TITLE
Narrator does not read subtext of ui fabric Close type Dialog in scan mode off

### ISSUE THREAD

**Issue by shpanwar** [OfficeDev/office-ui-fabric-react#11591]
### Environment Information

- **Package version(s)**:  office-ui-fabric-react@7.59.0
- **Browser and OS versions**: Edge, Windows 10

### Describe the issue:

When the scan mode is turned off, narrator does not annonce sub text of Close type Dialog(or isBlocking: false)

#### Actual behavior:
Repro steps:

1. Open Narrator.
2. Turn off scan mode by pressing Caps Lock + Space key
3. Use keyboard to navigate focus to the open the Dialog which is Close type or isBlocking: false
Press [Enter].
Observe that Narrator reads the dialog title, but does not read the dialog subtext. 
Is there some extra property that i need to add to the dialog so that it reads subtext as well?

#### Expected behavior:

Narrator is expected to read the dialog title and then subtext in when such Dialog is opened.

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Production

**Comment by jdhuntington** [comment]
Hi @shpanwar - The `Dialog` control provides some hooks for specifying additional content. Have a look at https://codepen.io/jdh-msft/pen/OJPOyKO?editors=0010#0 and see if the `Label` controls that contain `TODO` comments help resolve the issue. Unfortunately Narrator does not always read all of the expected content outside of scan mode, but this will definitely improve the functionality. It may require duplicating the subtext into an additional label for your case.
